### ISSUE TITLE
OIP 14 Prevent heating on weekends

### ISSUE THREAD

**Issue by ligi** [ethdevberlin/OIPS#14]
Just came in and no one was there but the thermostats where on 5.
Possible solutions:
 * electronic thermostats that prevent heating on weekends
 * make it part of the shutdown protocol to reduce heating when the last one leaves

**Comment by renaynay** [comment]
Is all heating already turned off for spring/summer?

**Comment by msooseth** [comment]
Yessss! And not just weekends, but nights as well :) It's important, weekends is only 28% of the time (2/7), but night is 50% of the time (8pm-8am)! So actually more savings, even if you keep it on during the weekends ;)

**Comment by msooseth** [comment]
There are in total the following regulators in the office:
* 4+4 on the kitchen side
* 1 in the elevator room
* 4+2 on the non-kitchen side
* 5 in the upper floor
So a total of 20.

[Here](https://www.amazon.de/Homexpert-Honeywell-HR25-Energy-Programmierbarer-Heizk%C3%B6rperthermostat/dp/B0BGRTY6FQ) is a version that's the simplest manual system, at 25 EUR/piece.

[Here](https://www.pearl.de/a-ZX5281-3101.shtml) is a remote-controlled Zigbee version at 36.6 EUR/piece.

If we go with the cheapest, it's 25*20=500 EUR, if we go with the smart, it's 755 EUR.

Above are **BALLPARK ONLY**, they are not meant to be recommendations.

**Comment by msooseth** [comment]
Some useful links from ligi, so we don't forget:

https://www.zigbee2mqtt.io/devices/SRTS-A01.html
https://www.zigbee2mqtt.io/devices/TS0601_thermostat.html
https://www.amazon.de/-/en/Radiator-Thermostat-Programmable-Compatible-Required/dp/B0B8GP5D8K

**Comment by ligi** [comment]
Would really like to get one with communication (ZigBee, WiFi, ..) - configuring them one by one is pain and having a communication protocol allows us more options later (like switching off the heating once the last one left the office - detected via the usage of the AccessPoints)

Also linking: https://github.com/ethdevberlin/OIPS/issues/27

**Comment by msooseth** [comment]
We discussed getting 2 of these + the ZigBee terminal:

Qiumi Zigbee Thermostat Radiator Valve, Smart Programmable Thermostat, Temperature Heater, eTRV, Alexa, Googlehome Voice Control, Plug Type is M30 x 1.5

Link: https://www.amazon.de/-/en/Thermostat-Radiator-Programmable-Temperature-Googlehome/dp/B09GYRK5GR/ref=sr_1_7?crid=PY6BXT2Y8I8J&keywords=thermostat%2Bzigbee&qid=1668782001&sprefix=thermostat%2Bzigbee%2Caps%2C87&sr=8-7&th=1

We should try them out and then buy more of the same. Let's get 3, so we can test out zones.

**Comment by kuzdogan** [comment]
What's the status of this? I think we really should get one

**Comment by msooseth** [comment]
I agree. I wanted to talk to Maurice, since he needs to approve anyway and Rose is leaving her position unfortunately. Maybe you can draft an email?

**Comment by raiatheofficefairy** [comment]
Hey, quick update - we're ordering 3 of [these](https://www.amazon.de/-/en/gp/product/B0B8GP5D8K/ref=ox_sc_act_title_1?smid=APO2Y9Q24H85O&th=1) + the hub to test them out :)

**Comment by ligi** [comment]
Great! Did you also order the ZigBee gateway for these?

**Comment by baumstern** [comment]
Prevent heating is also a good idea if
* When no one is using upstairs
* When everyone left the office (e.g. during night)
